index.html

```diff
@@ -12,8 +12,8 @@
     <header class="clearfix">
         <div class="logotype">FASHION</div>
 
-        <nav>
-           <a class="nav-link-active" href="#">Home</a>
+        <nav class="nav-menu">
+            <a class="nav-link-active" href="#">Home</a>
```

Comment:
Classes should be composing styles. Active link should have same class as other items + an `.active` class that adds specific styles.
```suggestion
            <a class="nav-link-active nav-link-active" href="#">Home</a>
```

Reply:
Also it's a good convention naming classes according to their hierarchy in some cases (like this one). Consider this pseudocode:
```
.nav-menu
   .nav-menu-item
   .nav-menu-item.active
```